src/redux/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const store = createStore(booksReducer, applyMiddleware(logger));
```

Comment:
- [ ] Kindly, update your store configuration by importing and invoking `combineReducers` to connect your store with multiple reducers other than the `booksReducer` (you might need to extend the store later for `Categories` and any other extra features).

Reply:
Hi @omar-labana thanks for your suggestion, I saw that requirement but I really taught

Reply:
Hi @omar-labana thanks for your suggestion, I understand that I have to use <code>combineReducers</code> to manage multiple Reducers but for this milestone is not necessary to makes the project work. Anyway I will update my project to complete the full requirements but I just wanted to share my opinion.

Reply:
@AlexRS90 Thanks for your understanding Alex, I had to request it because it's mentioned in the project requirements/ review guidelines. Sorry if it's a bit annoying😁.

Reply:
Thank you @omar-labana  I already submit the project 🧑‍💻  🤓